conductr_cli/conduct_main.py

```diff
@@ -386,6 +389,20 @@ def build_parser(dcos_mode):
 
     agents_parser.set_defaults(func=conduct_agents.agents)
 
+    # Sub-parser for `load-license` sub-command
+    load_license_parser = subparsers.add_parser('load-license',
+                                                help='Obtains license from Lightbend.com and '
```

Comment:
You need to set the new lines manually when using the `RawTextHelpFormatter`. So change it to:

```
help='Obtains license from Lightbend.com and \n'
     'loads the license into ConductR',
```